packages/typescript-configs/core.json

```diff
@@ -1,7 +1,6 @@
 {
-    "$schema": "https://json.schemastore.org/tsconfig",
     "compilerOptions": {
-        "moduleResolution": "bundler",
+        "moduleResolution": "nodenext",
```

Comment:
Why `nodenext` instead of `bundler` ?

Reply:
As mentionned in slack, nodenext should be use for all librairies, as we want to ensure the extensions are added in ESM library code.  Only web applications should use the module: bundler as we don't care if they add the extensions or not.